packages/web/hooks/limit-orders/use-place-limit.ts

```diff
@@ -378,33 +378,39 @@ export const usePlaceLimit = ({
     placeLimitMsg,
   ]);
 
-  const { data: balances, isLoading: isBalancesLoading } =
-    api.local.balances.getUserBalances.useQuery(
-      { bech32Address: account?.address ?? "" },
+  const { data, isLoading: isBalancesLoading } =
+    api.edge.assets.getUserAssets.useQuery(
+      {
+        userOsmoAddress: account?.address ?? "",
+      },
       {
         enabled: !!account?.address,
-        select: (balances) =>
-          balances.filter(
-            ({ denom }) =>
-              denom === baseAsset?.coinMinimalDenom ||
-              denom === quoteAsset?.coinMinimalDenom
+        select: ({ items }) =>
+          items.filter(
+            ({ coinMinimalDenom }) =>
+              coinMinimalDenom === baseAsset?.coinMinimalDenom ||
+              coinMinimalDenom === quoteAsset?.coinMinimalDenom
```

Comment:
Maybe it's a good idea to use toLowerCase to avoid case-sensitive mismatches. Thoughts?

Reply:
For both sides of the equality check?

Don't see much downside personally and should be safer